azext_iot/operations/dps.py

```diff
@@ -621,6 +658,11 @@ def _validate_allocation_policy_for_enrollment(allocation_policy,
                 raise CLIError('Please provide a hub to be assigned with device.')
             if iot_hub_list and len(iot_hub_list) > 1:
                 raise CLIError('Only one hub is required in static allocation policy.')
+        if allocation_policy == AllocationType.custom.value:
+            if webhook_url is None:
```

Comment:
I think we can check if either webhook_url or api_version is None or empty, then we provide the CLI error message which includes both arguments that must be included...rather than one at a time.

Reply:
Message could be something like "Please provide both the Azure function webhook url and provisioning service api-version when the allocation-policy is defined as Custom."

Reply:
Updated